packages/cicero-core/src/templateloader.js

```diff
@@ -80,7 +80,7 @@ class TemplateLoader extends FileLoader {
 
         // add model files
         Logger.debug(method, 'Adding model files to model manager');
-        template.getModelManager().addAPModelFiles(ctoModelFiles, ctoModelFileNames, true); // always offline
+        template.getModelManager().addAPModelFiles(ctoModelFiles, ctoModelFileNames, options && options.offline);
```

Comment:
This is an interesting breaking change. This means that it's not safe to assume that template archives are have all of their model dependencies resolved?

Reply:
Ah no that's a good point. Or a good question. I believe it is safe, but the `offline` flag is new and when I went over it again, I hadn't thought through the archive v directory distinction. I will revert this change.